src/query/storage/converter.go

```diff
@@ -182,17 +187,15 @@ func SeriesToPromSamples(series *ts.Series) []*prompb.Sample {
 	return samples
 }
 
-const (
-	// TODO(arnikola) get from config
-	initRawFetchAllocSize = 32
-)
-
 func iteratorToTsSeries(
 	iter encoding.SeriesIterator,
 	namespace ident.ID,
 ) (*ts.Series, error) {
 	if namespace == nil {
 		namespace = iter.Namespace()
+		if namespace == nil {
```

Comment:
don't want to fail request here ?

Reply:
Namespace isn't really used for anything as far as I know, so it's not particularly important; can get rid of it without any issues.